c182s.xml

```diff
@@ -1532,7 +1532,7 @@
     </external_reactions>
     
     <system file="c182s-stallhorn"/>
-    <system file="propulsion"/>
+  <system file="propulsion"/>
```

Comment:
@HHS81 btw how do you handle indentations in this project? I haven't figured out the policy from the code yet